android/src/main/java/com/applovin/reactnative/AppLovinMAXAdView.java

```diff
@@ -23,12 +21,106 @@
     private int       currentWidthPx;
     private int       currentHeightPx;
 
+    private String      adUnitId;
+    private MaxAdFormat adFormat;
+    private String      placement;
+    private String      customData;
+    private Boolean     adaptiveBannerEnabled;
+    private Boolean     autoRefresh;
+
     public AppLovinMAXAdView(final Context context)
     {
         super( context );
         this.reactContext = (ThemedReactContext) context;
     }
 
+    void setAdUnitId(final String value)
+    {
+        // Ad Unit ID must be set prior to creating MaxAdView
+        if ( adView != null )
+        {
+            AppLovinMAXModule.e( "Attempting to set Ad Unit ID " + adUnitId + " after MaxAdView is created" );
+            return;
+        }
+
+        adUnitId = value;
+
+        attachAdView();
+    }
+
+    void setAdFormat(final String value)
+    {
+        // Ad format must be set prior to creating MaxAdView
+        if ( adView != null )
+        {
+            AppLovinMAXModule.e( "Attempting to set ad format " + adUnitId + " after MaxAdView is created" );
+            return;
+        }
+
+        if ( "banner".equals( value ) )
+        {
+            adFormat = AppLovinMAXModule.getDeviceSpecificBannerAdViewAdFormat( reactContext );
+        }
+        else if ( "mrec".equals( value ) )
+        {
+            adFormat = MaxAdFormat.MREC;
+        }
+        else
+        {
+            AppLovinMAXModule.e( "Attempting to set an invalid ad format of \"" + value + "\" for " + adUnitId );
+            return;
+        }
```

Comment:
General convention - any logical change(s) I request on one platform should be brought over to the other platform (in this case, iOS).

Reply:
Right, my bad.   Updated the iOS version.